downstream/modules/platform/proc-aap-activate-with-credentials.adoc

```diff
@@ -1,11 +1,21 @@
 
-[id="proc-aap-activate-with-credentials_{context}"]
+[id="proc-aap-activate-with-credentials"]
 
 = Activate with credentials
 
 When {PlatformNameShort} launches for the first time, the {PlatformNameShort} Subscription screen automatically displays. You can use your Red Hat credentials to retrieve and import your subscription directly into {PlatformNameShort}.
 
-.Procedures
+[NOTE]
+====
+You are opted in for {Analytics} by default when you activate the platform on first time log in. This helps Red Hat improve the product by delivering you a much better user experience. You can opt out, after activating {PlatformNameShort}, by doing the following: 
+
+. From the navigation panel, select {MenuSetSystem}.
+. Uncheck the *Gather data for {Analytics}* option.
```

Comment:
Per IBM style guide -> Use "clear", not "uncheck" or "unselect", to refer to removing a mark from a check box.

Reply:
Fixed in https://github.com/ansible/aap-docs/pull/1883